slimta/edge/wsgi.py

```diff
@@ -158,17 +194,18 @@ def __init__(self, queue, hostname=None, validator_class=None,
                  uri_pattern=None):
         super(WsgiEdge, self).__init__(queue, hostname)
         self.validator_class = validator_class
-        if isinstance(uri_pattern, basestring):
+        if isinstance(uri_pattern, six.string_types):
             self.uri_pattern = re.compile(uri_pattern)
         else:
             self.uri_pattern = uri_pattern
 
     def __call__(self, environ, start_response):
-        self._trigger_ptr_lookup(environ)
+        unicode_environ = decode_environ(environ)
```

Comment:
Nitpick, but I'd suggest naming the function argument `environ_raw` and then having `environ = decode_environ(environ_raw)`, it would make usage a little bit cleaner I think.